plugins/nanospell/plugin.js

```diff
@@ -59,11 +59,29 @@
 		// (for example, p, li, td)
 		// and provides a mechanism for iterating over each word within,
 		// ignoring non-block elements.  (for example, span)
-		var isNotBookmark = CKEDITOR.dom.walker.bookmark(false, true);
+		var bookmarkGuardFn = CKEDITOR.dom.walker.bookmark(false, true);
```

Comment:
In a CKEDITOR dom walker, the "guard" function (it's an actual parameter) is a function that is called on every node to figure out if you should abort the walk.  

We wrap this function below in order to actually store the result, and I couldn't think of a good name for the original.  :/